src/theme/BlogPostItem/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { DiscussionEmbed } from 'disqus-react'
+import BlogPostItem from '@theme-original/BlogPostItem';
+import {useBlogPost} from '@docusaurus/plugin-content-blog/client'
+
+export default function BlogPostItemWrapper(props) {
+  const { metadata } = useBlogPost()
+  const { frontMatter, slug, title } = metadata
+  const { comments = true } = frontMatter
+  return (
+    <>
+      <BlogPostItem {...props} />
+      {comments && (
+        <DiscussionEmbed
+          shortname='https-replica-io-dev'
+          config={{
+            url: slug,
+            identifier: slug,
+            title,
+            language: 'en_US',
```

Comment:
I think we can remove this line. Or use `currentLocale` obtained through [`useDocusaurusContext`](https://docusaurus.io/docs/docusaurus-core#useDocusaurusContext).